addons/traefik/1.7.x/tests/00-assert.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: kubeaddons.mesosphere.io/v1beta1
```

Comment:
shouldn't be merged if the catalog is going to use this as addon repository
https://jira.d2iq.com/browse/D2IQ-68485